apps/www/lib/redirects.js

```diff
@@ -1901,4 +1901,9 @@ module.exports = [
     source: '/legal/soc2',
     destination: 'https://forms.supabase.com/soc2',
   },
+  {
+    permanent: true,
+    source: '/:ref/settings',
```

Comment:
Not certain if this is correct. Don't see a similar example in this file

Reply:
hmm yeah i dont think this should be here - this file's for the marketing site